features/step_definitions/quiet_steps.rb

```diff
@@ -0,0 +1,9 @@
+Then(/^(.+) (issues|violations) should not be printed to output$/) do |what, _|
+  executing_output = send(:"executing_#{what.gsub(' ', '_')}")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.